components/org.wso2.carbon.identity.oauth.dcr/src/main/java/org/wso2/carbon/identity/oauth/dcr/DCRClientException.java

```diff
@@ -22,6 +22,7 @@
 /**
  * Exception class used to handle DCR client related exceptions.
  */
+@Deprecated
```

Comment:
We need to add what can be used instead of the deprecated method. Check this in all the deprecated methods in this PR.

Reply:
This deprecation was done as a requirement in IS 6.0.0. It is recommended to use `/identity/oauth2/dcr/v1.1` instead.

Reply:
Resolved in https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1928